src/HotChocolate/Core/src/Authorization/AuthorizeAttribute.cs

```diff
@@ -65,7 +65,7 @@ public AuthorizeAttribute(ApplyPolicy apply)
     /// <summary>
     /// Specifies when the authorization directive shall be applied.
     /// </summary>
-    public ApplyPolicy Apply { get; set; } = ApplyPolicy.Validation;
+    public ApplyPolicy Apply { get; set; } = ApplyPolicy.BeforeResolver;
```

Comment:
@michaelstaib Do yo remember why the default ApplyPolicy changed to `BeforeResolver`?  Should this change to the default behavior be added to the breaking changes list?
https://chillicream.com/docs/hotchocolate/v15/migrating/migrate-from-13-to-14

Reply:
The only reason I am asking is that the YouTube video [linked here](https://chillicream.com/docs/hotchocolate/v15/security/authorization) says that `ApplyPolicy.Validation` is the default that was set with v13.